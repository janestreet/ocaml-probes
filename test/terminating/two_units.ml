let () =
  T.test ();
  S.test ();
  ()
;;
