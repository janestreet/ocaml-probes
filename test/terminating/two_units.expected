From T: foo
From T: foo was true
From T: foo
From T: foo was true
